server/controllers/chartsController.js

```diff
@@ -22,6 +23,22 @@ const postProcessData = (data, studyTotals) => {
     const percent = studyCountsToPercentage(count, totals)
     const existingEntriesForStudy = processedDataBySite.get(study)
 
+    if (!!targetValue && study !== TOTALS_STUDY) {
+      if (totalsValueTargets[valueLabel]) {
+        totalsValueTargets[valueLabel] =
+          +totalsValueTargets[valueLabel] + +targetValue
+      } else {
+        totalsValueTargets[valueLabel] = +targetValue
+      }
+    } else {
+      if (totalsValueTargets[valueLabel]) {
+        totalsValueTargets[valueLabel] =
+          +totalsValueTargets[valueLabel] + +totalsForStudy.count
+      } else {
+        totalsValueTargets[valueLabel] = +totalsForStudy.count
+      }
```

Comment:
do we need to make sure this isn't the totals study in the `else` condition?

Reply:
Thank you for that point. 
https://github.com/AMP-SCZ/dpdash/pull/206/commits/86de73a816b9428080e1b1bed237c9fdf39e27c6